packages/fuel-indexer-schema/src/lib.rs

```diff
@@ -78,176 +74,10 @@ pub enum FtColumn {
     UInt16(Option<UInt16>),
     UInt4(Option<UInt4>),
     UInt8(Option<UInt8>),
+    NoRelation(Option<NoRelation>),
 }
 
 impl FtColumn {
-    pub fn new(ty: ColumnType, size: usize, bytes: &[u8]) -> FtColumn {
```

Comment:
@deekerno If you have this commit up, I can cherry-pick it from you (I think you might get a conflict regardless what we do 🤔)